pom.xml

```diff
@@ -150,6 +195,223 @@ under the License.
       <version>2.0.0.0</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.opensaml</groupId>
+      <artifactId>opensaml-core</artifactId>
+      <version>3.4.3</version>
+      <exclusions>
+        <exclusion>
+          <groupId>org.slf4j</groupId>
+          <artifactId>slf4j-api</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>org.slf4j</groupId>
+          <artifactId>jcl-over-slf4j</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>commons-codec</groupId>
+          <artifactId>commons-codec</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework</groupId>
```

Comment:
wrong dependency pac4j-saml uses spring-core 5.0.4

Reply:
and here is the big problem the Jenkins core has spring 2.5.6, so add a dependency to spring 5.0.4 is a highly risky decision that affects other plugins and Jenkins Core, I've remembered that was was the cause I did not move to pac4j 2.0.0 I have plans to move everything to use openSAML without pac4j but it will not be soon.